ui/src/AppCharting/LineChart.svelte

```diff
@@ -36,13 +45,18 @@
         },
         position: 'right'
       },
-      onrendered: onRendered,
+      oninit: () => onRendered(theKey),
       size: {height: height}
       // zoom: {enabled: true}
     });
   }
 
-  function onRendered(): void {
+  function onRendered(theKey: any): void {
+    if (theKey !== key) {
+      console.debug("There's a different ref in town, skipping", theKey, key);
```

Comment:
Will this be relatively infrequent? I don't know the difference between `console.log()` and `console.debug()`, but presumably they're log levels you can filter based on? Or maybe you have to set something special in the browser to see debug messages?

Reply:
I don't think this really happens any more. `debug` === `log` just a little more descriptive in clode.  It use to show up in a blue box (like error do in red) but it's been a while since that was true.